src/amplitude-client.js

```diff
@@ -1552,4 +1598,33 @@ if (BUILD_COMPAT_2_0) {
  */
 AmplitudeClient.prototype.__VERSION__ = version;
 
+/**
+ * Determines whether or not to push call to this._q or invoke it
+ * @private
+ */
+AmplitudeClient.prototype._shouldDeferCall = function _shouldDeferCall() {
+  return this._pendingReadStorage || this._initializationDeferred;
```

Comment:
what is pending read storage?

Reply:
we use this with `AsyncStorage` (react-native) to batch calls

Reply:
Actually, looking closer to the code, it's not only used for react-native
cc: @blazzy

Reply:
It's only used for async storage yeah.

Reply:
👍